foxsec-slack-bot/slackutils.go

```diff
@@ -18,6 +18,23 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+func checkUsersGroups(email string) (bool, error) {
+	person, err := globalConfig.personsClient.GetPersonByEmail(email)
```

Comment:
Should we add some validation of the format of `email` here since this value gets passed directly as a URL parameter later?

Reply:
Yeah absolutely